docs/pages/setup/admin/daemon.mdx

```diff
@@ -31,117 +35,174 @@ $ readlink /sbin/init
+     Active: inactive (dead)
+```
 
-Copy the recommended Teleport unit file for `systemd` below and paste the
-contents into a file called `/usr/lib/systemd/system/teleport.service`:
+The contents of the default unit file will reflect the following defaults:
 
-```systemd
+```code
 [Unit]
 Description=Teleport SSH Service
 After=network.target
 
 [Service]
 Type=simple
 Restart=on-failure
-ExecStart=/usr/local/bin/teleport start --config=/etc/teleport.yaml --pid-file=/run/teleport.pid
+EnvironmentFile=-/etc/default/teleport
+ExecStart=/usr/local/bin/teleport start --pid-file=/run/teleport.pid
 ExecReload=/bin/kill -HUP $MAINPID
 PIDFile=/run/teleport.pid
+LimitNOFILE=8192
 
 [Install]
 WantedBy=multi-user.target
 ```
 
-<Notice type="tip">
+Users who rely on DEB or RPM can enter the following commands to enable and start the systemd `teleport` daemon at any time. Enabling the daemon using `systemctl` allows systemd to place the daemon in it's dependancy tree, ensuring that the teleport service will be automatically restarted on any potential reboot:
 
-If `/usr/lib/systemd/system/` does not exist, consult the list of
-[unit file load paths](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
-for other supported paths.
+```code
+$ sudo systemctl enable teleport.service
+```
 
-</Notice>
+Next, the following command will officially start the teleport service daemon:
 
-Enable the unit so systemd can place it in its dependency tree:
+```code
+$ sudo systemctl start teleport.service
+```
+
+Confirm that the Teleport daemon was installed correctly by checking the service status. The following command should output a status of `Active: active (running)`:
 
 ```code
-$ sudo systemctl enable teleport.service
-Created symlink /etc/systemd/system/multi-user.target.wants/teleport.service → /lib/systemd/system/teleport.service.
+$ sudo systemctl status teleport.service | grep Active
+Active: active (running) since Mon 2022-04-18 18:33:41 UTC; 41s ago
+```
+
+### Creating the systemd Daemon When Installed From Source
+
+Systemd is designed to use [unit files](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) which are the primary file used to define and set the conditions for
+any needed service daemon on systemd, in this case the `teleport` service. In order to more easily support systemd configurations, Teleport is capable of recommending and assisting with the installation of a default 
+unit file using as little as one command (`teleport install systemd -o /unit/file/load/path`).
+
+The following teleport command will output a [systemd unit file](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) (using `stdout`) to the terminal to be reviewed and used as needed:
+
+```code
+$ teleport install systemd
 ```
 
-Start the unit:
+The output will resemble the following:
 
 ```code
-$ sudo systemctl start teleport.service
+[Unit]
```

Comment:
Same comment as above - I'd prefer to not have this snippet pasted in a bunch of different places.

Reply:
This has been removed